src/control/reverse_interface.cpp

```diff
@@ -55,8 +61,19 @@ bool ReverseInterface::write(const vector6d_t* positions, const comm::ControlMod
   uint8_t buffer[sizeof(int32_t) * MAX_MESSAGE_LENGTH];
   uint8_t* b_pos = buffer;
 
-  // The first element is always the keepalive signal.
-  int32_t val = htobe32(keepalive_count_);
+  int read_timeout = robot_receive_timeout.verifyRobotReceiveTimeout(comm::ControlMode::MODE_FREEDRIVE, step_time_);
```

Comment:
Is there a reason we're not using the actual `control_mode` here?

Reply:
No, that was a copy paste mistake by me...